File: ws/salazar/u2/ws32PQUIZ/ChickenFarmer.py
import Coop
class ChickenFarmer:
    def __init__(self,id,coops):
        self.id = int(id)
        self.coops = Coop(coops)
        
    def add(self,coop):
        self.coops.append(coop)
        
    def remove(self,coopId):
        for coop in self.coops:
            if(coop.id==coopId):
                self.coops.pop(coop)
                
    def resetIteration(self):
        self.coops.clear()
        
    def next():
        return Coop

File: ws/suquillo/u2/ws32Pquiz/ChickenFarmSimulator/ChickenFarmSimulator/ChickenFarmModel/ChickenFarmer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ChickenFarmSimulator.ChickenFarmModel
{
    internal class ChickenFarmer
    {
        public String name { get; set; }
        public Coop[] coops { get; set; }  
        public void add(Coop coop) { }
        public void remove(int coopId) { }
        public void resetIteration() { }
        public Coop next()
        {
            return new Coop();
        }
    }
}
